limDaSeul/db/migrations/20221124130001_users.sql

```diff
@@ -0,0 +1,13 @@
+-- migrate:up
+CREATE TABLE users(
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(50) NOT NULL,
+    email VARCHAR(100) NOT NULL,
+    profile_image VARCHAR(500) NOT NULL,
+    password VARCHAR(1000) NOT NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
```

Comment:
다슬님, update_at의 경우 수정사항이 발생했을 때의 데이터 값입니다. 해당사항을 NOT NULL로 지정하게 되었을 때 오류가 발생하게되니 아래와 같이 수정해 주세요
```suggestion
updated_at TIMESTAMP NULL DEFAULT NULL ON UPDATE CURRENT_TIMESTAMP
```